test/Microsoft.AspNetCore.Server.Kestrel.Performance/global.json

```diff
@@ -0,0 +1,3 @@
+﻿{
+  "projects": [ "..\\" ]
```

Comment:
BenchmarkDotNet crawls the directory tree till it finds `global.json` or `.sln` so is to get it to stop in correct place, otherwise it can't find the `Microsoft.AspNetCore.Server.Kestrel.Performance` project